gadget/params.c

```diff
@@ -312,6 +312,21 @@ create_gadget_parameter_set()
     param_declare_double(ps, "NuPartTime", OPTIONAL, 0.3333333, "Scale factor at which to turn on hybrid neutrino particles.");
     /*End parameters for the massive neutrino model*/
 
+    /*Parameters for the Excursion Set Algorithm*/
+    param_declare_int(ps, "ExcursionSetFlag", OPTIONAL, 0, "Use the excursion set instead of the global UV field");
+    param_declare_int(ps, "UVBGdim", OPTIONAL, 64, "Side length of the excursion set grid.");
+    param_declare_int(ps, "UVBGSaveGrids", OPTIONAL, 1, "Flag to save reionisation grids.");
```

Comment:
Does this need to be an option or can we just leave it always on?

Reply:
If we always want to save the grids on every output step I can remove it. I put it there along the same lines as the outputFOF flag in case we only wanted the particle info for some runs.

Reply:
Do I understand right that the saved grids are needed to resume from the snapshot? In which case I think we would always want them when the ExcursionSetFlag is on?

Reply:
That's true, I don't think I considered the checkpointing when making this flag. It's been removed.

Reply:
Thanks